### DISCUSSION TITLE
Date time picker

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#11364] [CLI - PWA mode]
Hello Friends ,

Am try to Create two date time picker which is Date FROM and Date TO  which is separated by an arrow .Both the date and the time should be inside the input field.
but am not able to add in both inside the input field and to include the arrow.

Please the my code below https://codepen.io/akashramtohul4/pen/vYJQbXO?editors=101

here is an image how it should look like 
![Capture](https://user-images.githubusercontent.com/84211129/141972871-53f0559c-c607-4360-806a-450122f2651b.PNG)

Can someone Please help

Thanks

**Comment by pdanpdan** [comment]
You need to put there 2 input fields and an icon with arrow in a div with class row.
And use different models for from and to.
Try to play you the codepen you cloned and let's see what is the problem you cannot solve.

**Reply by akashramtohul4** [reply]
hello padanpdan 
am unable to add it in the input field when i add it in the div its mess-up and making the arrow appear

**Comment by pdanpdan** [comment]
I cannot see any change from the original example in your codepen.

**Reply by akashramtohul4** [reply]
https://codepen.io/akashramtohul4/pen/vYJQbXO?editors=101

Here is the link

**Comment by pdanpdan** [comment]
You know, it would have helped a lot if you would have put the slightest effort to do something - it was not working because:
- you had missing commas in your JS
- you did not bother to rename the model used in the popups when you copied it
- you had self closing tags in HTML

https://codepen.io/pdanpdan/pen/porqEvj?editors=1010
